### PR TITLE
Announce the CfP

### PR DESCRIPTION
**Pull request by badboy**

### CODE REVIEW

File: _posts/2019-07-22-cfp-for-barcelona.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+We're currently building up this year's CfP committee. Stay tuned for more news soon.
+
+## CfP for workshops
+
+We're also looking for some workshops.
+With the changed format of the conference, we will have a few more, but shorter workshops.
+We are looking for people hosting workshops on interesting topics all around Rust.
+
+Workshops will be happening in the afternoon of both Saturday and Sunday.
+Therefore workshops can be on a single afternoon, can be repeated on the second day or can potentitally stretch across both days.
```
Comment (GuillaumeGomez) [inline _posts/2019-07-22-cfp-for-barcelona.md]: "potentitally"

File: _posts/2019-07-22-cfp-for-barcelona.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+Please let us know what is suitable for your proposal.
```
Comment (GuillaumeGomez) [inline _posts/2019-07-22-cfp-for-barcelona.md]: "Please let us know what is (**more**?) suitable for your proposal."

**Comment by GuillaumeGomez** [comment]
Looks good to me!

File: _posts/2019-07-22-cfp-for-barcelona.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+Starting today the [Call for Proposals](https://cfp.rustfest.eu/events/rustfest-barcelona-2019) is open!
+
+We would love to hear your proposal for a talk. Make sure to submit before **August 19, 23:59 UTC**!
+As always, we'll be prioritizing talks not yet recorded at Rust conferences.
+
+You are free to talk about anything Rust-related, come and share your own personal Rust experience!
+From code, to community and not forgetting experiences in production, best practices, theory or anything in between. We are happy to hear it from you.
+
+First time speaking? Submit! It's totally okay! If we accept your talk in our (blind) selection procedure we're happy to be your first experience. If you'd like, we're also happy to mentor you to give you some tips.
```
Comment (apiraino) [inline _posts/2019-07-22-cfp-for-barcelona.md]:
```suggestion
First time speaking? Submit! It's totally okay! If your talk will be selected in our (blind) selection procedure, we'll be happy for Rustfest to be your first experience on stage. We're also happy to mentor you with tips, guidelines and review your talk.
```

Comment (apiraino) [reply _posts/2019-07-22-cfp-for-barcelona.md]: I would also evaluate a more clear wording for "stage newcomers" to review their talk. We might want every talk to be as refined, balanced and shining as possible.

**Review by RafaelaWetzel — COMMENTED** [review]
Looks fine to me as well!

Noticed a few things in the[ CFP app](http://cfp.rustfest.eu/events/rustfest-barcelona-2019) text though:

1. Topics for Rustfest: It says "Given the _one-day nature_ of the event, ..." > **two days**
2. The Perks: "-If you instead want to stay with fellow Rustaceans, get in touch.
-The same goes for travel information! 
If you need travel information or would like to stay with fellow Rustaceans, get in touch with us, we can help you out!" > **Double mention**
3. Guidelines: "All talks are in English. If English is not _our_ first (or even second) language that is totally okay, our attendees understand and are supportive." > **your**

**Comment by badboy** [comment]
Thanks, @RafaelaWetzel, addressed those.

**Pull request merged**